mne/data/coil_def.dat

```diff
@@ -510,72 +468,127 @@
  -0.1250  4.082e-03 -7.071e-03  5.000e-02  0.000  0.000  1.000
  -0.1250 -4.082e-03  7.071e-03  5.000e-02  0.000  0.000  1.000
  -0.1250 -4.082e-03 -7.071e-03  5.000e-02  0.000  0.000  1.000
-2   7501   0    2  1.486e-02  5.740e-02 "Artemis123 axial gradiometer size = 1.486e-02 mm base = 5.740e-02 mm"
+0   7501    0   2  1.486e-02  5.740e-02	"Artemis system gradiometer size = 14.86  mm base = 57.40  mm"
```

Comment:
IIUC, class type (first entry) used to be 2 (axial gradiometer) but now has been changed to 0, which is not listed as a valid type at the top of the file...?